modules/ROOT/pages/manage-exchange-api-task.adoc

```diff
@@ -44,13 +72,19 @@ For example, if an API is defined with the `Implementation URI` set to `http://j
 |`/` | `https://domain.cloudhub.io/users` | `http://jsonplaceholder.typicode.com/users`
 | `/api` | `https://domain.cloudhub.io/api/users` | `http://jsonplaceholder.typicode.com/users`
 |===
-. Optionally, expand the *Advanced options* section and specify additional details:
-* Specify the validations and an API instance label for the API.
-* If you are creating an endpoint with proxy, also specify a scheme (HTTP or HTTPS) and a port, or accept the defaults.
-* Specify a response timeout, if required. 
+.. Optionally, expand the *Advanced options* section and specify additional details:
```

Comment:
See comments at Line 48.

I wonder if these options sections are labeled intuitively enough for us to skip these bullet lists in favor of a step that simply states something like...:

"If you want to specify additional details, expand and complete *Advanced options*."

Reply:
Done! Thanks!